recipe/meta.yaml

```diff
@@ -28,7 +28,8 @@ requirements:
     - python
     - setuptools
     - cython
-    - numpy
+    - numpy 1.13.3  # [py<37]
+    - numpy 1.14.5  # [py>=37]
```

Comment:
@isuruf and @rgommers these are the minimum versions we have that satisfy `scipy`. However, I'm would like to propose an update to `conda-forge`'s global pinning to `numpy 1.14` (more on that in our next core meeting).

Reply:
Seems like a good idea. NumPy 1.17 will be out in a month or so, and maintaining support for >4 numpy versions seems like a bit of a waste of resources.